app/views/fae/application/_header.slim

```diff
@@ -3,10 +3,10 @@ header.main-header#js-main-header
   h1: a href=root_path = @option.title
 
   - if @fae_topnav_items.present?
-    nav.js-nav
-      ul.main-header-nav
+    nav
+      ul.main-header-nav.js-nav data-coordinates=@fae_navigation.coordinates[0..1].join(',')
```

Comment:
Instead of getting JS involved, why not just use the index of the loop and compare it to the coordinate of the corresponding level? If they are the same you add the `-current` class via Ruby.